lib/parser/javascript.js

```diff
@@ -136,9 +129,6 @@ ReplyParser.prototype._parseResult = function (type) {
                 throw new IncompleteReadBuffer("Wait for more data.");
             }
             res = this._parseResult(ntype);
-            if (res === undefined) {
```

Comment:
This is dead code by changing the [`return undefined;`] (https://github.com/NodeRedis/node_redis/pull/852/files#diff-778060651e24292442eb47a196897e2eL95) to `return null;`